<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CropImagesWidget</class>
 <widget class="QWidget" name="CropImagesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CropImagesWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Range 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Range 5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>Range 6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_8">
         <property name="text">
          <string>Range 8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="crop">
       <property name="text">
        <string>Crop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save Cropped Images</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>crop</sender>
   <signal>clicked()</signal>
   <receiver>CropImagesWidget</receiver>
   <slot>crop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>clicked()</signal>
   <receiver>CropImagesWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>crop()</slot>
  <slot>loadCalib()</slot>
  <slot>loadRecons()</slot>
  <slot>save()</slot>
 </slots>
</ui>
